src/xor_space/xor_name.rs

```diff
@@ -271,7 +271,6 @@ mod tests {
     use super::*;
     use crate::unwrap;
     use bincode::{deserialize, serialize};
-    use rand;
```

Comment:
suprised it was not soptted by the clippy previously.

Reply:
maybe the particular lint just wasn't enabled before v1.43?

Reply:
yeah, maybe.